translations/ai_rules.tex/pl.po

```diff
@@ -195,12 +195,12 @@ msgid ""
 "If an AI Hero is instructed to draw a card, they will draw and resolve \\textbf{another card} from the AI Deck.\n"
 "\n"
 msgstr ""
-"Kiedy Bohater AI \\textbf{Aktywuje} Jednostkę, dobierz Kartę AI \\index{Karta AI} i rozpatrz ją zanim Jednostka wykona ruch lub zaatakuje.\n"
-"Jeżeli Talia AI wyczerpie się podczas Walki, nie dobieraj Kart.\n"
-"Efekt każdej Karty AI zależy od \\hyperlink{Difficulty}{Poziomu Trudności} gry.\n"
-"Karta Mocy \\includesvg[height=10px]{\\svgs/might.svg} zostaje przy Jednostce do momentu gdy nastąpi pierwszy atak/obrona.\n"
-"Talia Zaklęć AI jest używana za każdym razem, gdy dobierana jest Karta Magii \\includesvg[height=10px]{\\svgs/magic.svg} z Talii AI.\n"
-"Jeżeli Bohater AI otrzyma polecenie dobrania Karty, dobierze i rozpatrzy \\textbf{kolejną Kartę} z Talii AI.\n"
+"Kiedy Bohater AI \\textbf{Aktywuje} Jednostkę, dobierz Kartę AI \\index{Karta AI} i rozpatrz ją przed aktywacją Jednostki.\n"
+"Jeżeli Talia AI skończy się podczas Walki, nie dobieraj Kart.\n"
```

Comment:
```suggestion
"Jeżeli Talia AI skończy się podczas Walki, nie dobieraj z niej Kart.\n"
```

Reply:
tu nie wiem, jak odnieść się do sugestii - patrząc na PL instrukcję oraz na angielski tekst ja go rozumiem tak, że nie należy odnawiać tej talii kart poprzez ponowne potasowanie talii i utworzenie stosu na nowo. A skoro stos się skończył to już nie dobiera się kart... słowo "niej" po prostu nie ma potrzeby istnienia w tym miejscu ;)

Reply:
Jak dla mnie bez "z niej" brzmi zbyt ogólnie. Talia AI się skończyła? To gracz też nie dobiera. Tak to można zrozumieć, a to nieprawda.

Reply:
więc... ma być "z niej", według Ciebie @qwrtln?

Reply:
Tak, bo talii jest wiele.